R/github.R

```diff
@@ -116,7 +138,10 @@ use_github <- function(organisation = NULL,
       org = organisation,
       name = repo_name,
       description = repo_desc,
-      private = private,
+      visibility = visibility,
+      # this is necessary to set `visibility` in GHE 2.22 (but not in 3.2)
+      # hopefully it's harmless when not needed
+      .send_headers = c(Accept = "application/vnd.github.nebula-preview+json"),
```

Comment:
I should be using `.accept` here. Either that argument is new (to me) or I just forgot it existed.